### PR TITLE
Added splash, login design

### PR DESCRIPTION
**Pull request by dannylui91**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -18,14 +18,15 @@
...
-        <activity android:name=".login.SplashScreen" />
-        <activity android:name=".login.StartActivity">
+        <activity android:name=".login.SplashScreen" >
 
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
-            </intent-filter>
+            </intent-filter></activity>
```
Comment (jrodbx) [inline app/src/main/AndroidManifest.xml]: Command-A to select all.  Option-Command-L to format.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -18,14 +18,15 @@
...
+
+        <activity android:name=".login.StartActivity">
         </activity>
```
Comment (jrodbx) [inline app/src/main/AndroidManifest.xml]: use the single tag form: <activity android:name=".login.StartActivity"/>

File: app/src/main/res/layout/activity_splash_screen.xml [hunk 3]
```diff
@@ -4,10 +4,7 @@
     android:id="@+id/activity_splash_screen"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:paddingBottom="@dimen/activity_vertical_margin"
-    android:paddingLeft="@dimen/activity_horizontal_margin"
-    android:paddingRight="@dimen/activity_horizontal_margin"
-    android:paddingTop="@dimen/activity_vertical_margin"
-    tools:context="com.example.andresarango.aughunt.login.SplashScreen">
+    tools:context="com.example.andresarango.aughunt.login.SplashScreen"
+    android:background="@drawable/splash_bg">
```
Comment (jrodbx) [inline app/src/main/res/layout/activity_splash_screen.xml]: switch these two lines

File: app/src/main/java/com/example/andresarango/aughunt/login/SplashScreen.java [hunk 4]
```diff
@@ -11,5 +13,16 @@
...
+
+        Handler handler = new Handler();
+        handler.postDelayed(new Runnable() {
+            @Override
+            public void run() {
+                System.out.println("SPLASHING WAITING... 5s");
+                startActivity(new Intent(SplashScreen.this, StartActivity.class));
+                finish();
+            }
+        }, 5000);
```
Comment (jrodbx) [inline app/src/main/java/com/example/andresarango/aughunt/login/SplashScreen.java]: what happens if the user hits the back button before the 5 seconds are over? ;-)
hint: you probably want to move this to onStart and call handler.removeCallbacks() in onStop.

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (dannylui91) [reply app/src/main/AndroidManifest.xml]: Will do, but I got lazy when moving my intent filters around :(

File: app/src/main/java/com/example/andresarango/aughunt/login/SplashScreen.java (hunk 4 above)
Comment (dannylui91) [reply app/src/main/java/com/example/andresarango/aughunt/login/SplashScreen.java]: oh no don't do that :( , does handler.removeCallbacks() prevent the app from closing on back press?

**Review by Mila2010 — COMMENTED** [review]
Have reviewed Login screen code

**Review by Mila2010 — COMMENTED** [review]
Reviewed Login screen code

**Review by Mila2010 — COMMENTED** [review]
Left the comment already

**Review by cito125 — APPROVED** [review]
LGTM, cool icons

**Pull request merged**